examples/sagemaker/endpoint.yaml

```diff
@@ -0,0 +1,100 @@
+---
+apiVersion: sagemaker.aws.upbound.io/v1beta1
+kind: Endpoint
+metadata:
+  annotations:
+    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
```

Comment:
```suggestion
    meta.upbound.io/example-id: sagemaker/v1beta1/endpoint
    upjet.upbound.io/manual-intervention: "This resource is skipping because the EndpointConfiguration resource on which this resource depends was skipped."
```